docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command: >
+      sh -c "python manage.py wait_for_db && python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
```

Comment:
The command includes a custom management command `wait_for_db`, which is a good practice to ensure the database is ready before running migrations and starting the server.